src/main/resources/config-schema.yaml

```diff
@@ -1,43 +1,8 @@
-hbase.zoookeepers: "localhost"
+hbase.zookeepers: "localhost"
 
-hbase.catalog: "dengDIDataStore"
+hbase.catalog: "weatherdata"
```

Comment:
Feature name as catalog

Reply:
Catalog should be common for all the features right?

Reply:
As we discussed earlier?